azurerm/internal/services/containers/kubernetes_nodepool.go

```diff
@@ -133,6 +133,11 @@ func SchemaDefaultNodePool() *schema.Schema {
 					Computed:     true,
 					ValidateFunc: validation.StringIsNotEmpty,
 				},
+				"proximity_placement_group_id": {
+					Type:         schema.TypeString,
+					Optional:     true,
+					ValidateFunc: validation.StringIsNotEmpty,
```

Comment:
the `compute/validate` package (`github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/services/compute/validate`) exposes a method of validating this is a ProximityPlacementGroupID - could we use that here:

```suggestion
					ValidateFunc: computeValidate.ProximityPlacementGroupID,
```

Reply:
Oh, I looked for it, but apparently overlooked. Apologies.